packages/node/src/integrations/anr/index.ts

```diff
@@ -131,7 +131,7 @@ async function _startWorker(client: NodeClient, _options: Partial<Options>): Pro
 
   const timer = setInterval(() => {
     try {
-      const currentSession = getCurrentScope().getSession();
+      const currentSession = getCurrentHub().getScope().getSession();
```

Comment:
hmm we shouldn't do this 😅 was this done on purpose?

Reply:
I must have changed it back incorrectly after first accessing `getIsolationScope` here. fixed in [24181c7](https://github.com/getsentry/sentry-javascript/pull/10054/commits/24181c7f5f915b234517dd3b4911c0f66a3443bb)